{"text": "2021-05-08T15:40:46.989782+0000 ERROR An error has been caught in function 'dispatch', process 'MainProcess' (275764), thread 'Thread-5' (139990356977408):\nTraceback (most recent call last):\n\n  File \"/usr/lib/python3.8/threading.py\", line 890, in _bootstrap\n    self._bootstrap_inner()\n    \u2502    \u2514 <function Thread._bootstrap_inner at 0x7f52162d94c0>\n    \u2514 <Thread(Thread-5, started daemon 139990356977408)>\n  File \"/usr/lib/python3.8/threading.py\", line 932, in _bootstrap_inner\n    self.run()\n    \u2502    \u2514 <function Thread.run at 0x7f52162d91f0>\n    \u2514 <Thread(Thread-5, started daemon 139990356977408)>\n  File \"/usr/lib/python3.8/threading.py\", line 870, in run\n    self._target(*self._args, **self._kwargs)\n    \u2502    \u2502        \u2502    \u2502        \u2502    \u2514 {}\n    \u2502    \u2502        \u2502    \u2502        \u2514 <Thread(Thread-5, started daemon 139990356977408)>\n    \u2502    \u2502        \u2502    \u2514 (<socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('...\n    \u2502    \u2502        \u2514 <Thread(Thread-5, started daemon 139990356977408)>\n    \u2502    \u2514 <bound method ThreadingMixIn.process_request_thread of <django.core.servers.basehttp.WSGIServer object at 0x7f52105bc220>>\n    \u2514 <Thread(Thread-5, started daemon 139990356977408)>\n  File \"/usr/lib/python3.8/socketserver.py\", line 650, in process_request_thread\n    self.finish_request(request, client_address)\n    \u2502    \u2502              \u2502        \u2514 ('127.0.0.1', 48902)\n    \u2502    \u2502              \u2514 <socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <function BaseServer.finish_request at 0x7f5215011e50>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f52105bc220>\n  File \"/usr/lib/python3.8/socketserver.py\", line 360, in finish_request\n    self.RequestHandlerClass(request, client_address, self)\n    \u2502    \u2502                   \u2502        \u2502               \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f52105bc220>\n    \u2502    \u2502                   \u2502        \u2514 ('127.0.0.1', 48902)\n    \u2502    \u2502                   \u2514 <socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <class 'django.core.servers.basehttp.WSGIRequestHandler'>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f52105bc220>\n  File \"/usr/lib/python3.8/socketserver.py\", line 720, in __init__\n    self.handle()\n    \u2502    \u2514 <function WSGIRequestHandler.handle at 0x7f521228be50>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f52104ced60>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 172, in handle\n    self.handle_one_request()\n    \u2502    \u2514 <function WSGIRequestHandler.handle_one_request at 0x7f521228bee0>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f52104ced60>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 197, in handle_one_request\n    handler.run(self.server.get_app())\n    \u2502       \u2502   \u2502    \u2502      \u2514 <function WSGIServer.get_app at 0x7f521228b430>\n    \u2502       \u2502   \u2502    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f52105bc220>\n    \u2502       \u2502   \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f52104ced60>\n    \u2502       \u2514 <function BaseHandler.run at 0x7f52123050d0>\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f52104ce700>\n  File \"/usr/lib/python3.8/wsgiref/handlers.py\", line 137, in run\n    self.result = application(self.environ, self.start_response)\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2502    \u2514 <function BaseHandler.start_response at 0x7f5212305430>\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f52104ce700>\n    \u2502    \u2502        \u2502           \u2502    \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n    \u2502    \u2502        \u2502           \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f52104ce700>\n    \u2502    \u2502        \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f5211a803d0>\n    \u2502    \u2514 None\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f52104ce700>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/contrib/staticfiles/handlers.py\", line 76, in __call__\n    return self.application(environ, start_response)\n           \u2502    \u2502           \u2502        \u2514 <bound method BaseHandler.start_response of <django.core.servers.basehttp.ServerHandler object at 0x7f52104ce700>>\n           \u2502    \u2502           \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n           \u2502    \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f52159f9be0>\n           \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f5211a803d0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/wsgi.py\", line 133, in __call__\n    response = self.get_response(request)\n               \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function BaseHandler.get_response at 0x7f521232ff70>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f52159f9be0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 130, in get_response\n    response = self._middleware_chain(request)\n               \u2502    \u2502                 \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f5214e539d0>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f52159f9be0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f52159ebd90>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f5214e53a60>\n               \u2502           \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f52159ebd90>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f5214e56070>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f5214e53af0>\n               \u2502           \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f5214e56070>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.common.CommonMiddleware object at 0x7f5214e560d0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f5214e53b80>\n               \u2502           \u2514 <django.middleware.common.CommonMiddleware object at 0x7f5214e560d0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f5214e56100>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f5214e53c10>\n               \u2502           \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f5214e56100>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f5214e56310>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f5214e53dc0>\n               \u2502           \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f5214e56310>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f5214e562e0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f5214e53e50>\n               \u2502           \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f5214e562e0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f5214e563a0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function BaseHandler._get_response at 0x7f5214e53550>\n               \u2502           \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f5214e563a0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <bound method BaseHandler._get_response of <django.core.handlers.wsgi.WSGIHandler object at 0x7f52159f9be0>>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 181, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n               \u2502                \u2502         \u2502                \u2514 {}\n               \u2502                \u2502         \u2514 ()\n               \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <function ClientRecordView at 0x7f52119fd280>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 70, in view\n    return self.dispatch(request, *args, **kwargs)\n           \u2502    \u2502        \u2502         \u2502       \u2514 {}\n           \u2502    \u2502        \u2502         \u2514 ()\n           \u2502    \u2502        \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2502    \u2514 <function View.dispatch at 0x7f5214e64a60>\n           \u2514 <email_send.views.ClientRecordView object at 0x7f5210350c70>\n> File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 98, in dispatch\n    return handler(request, *args, **kwargs)\n           \u2502       \u2502         \u2502       \u2514 {}\n           \u2502       \u2502         \u2514 ()\n           \u2502       \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2514 <bound method ClientRecordView.post of <email_send.views.ClientRecordView object at 0x7f5210350c70>>\n  File \"/usr/lib/python3.8/contextlib.py\", line 75, in inner\n    return func(*args, **kwds)\n           \u2502     \u2502       \u2514 {}\n           \u2502     \u2514 (<email_send.views.ClientRecordView object at 0x7f5210350c70>, <WSGIRequest: POST '/contact/customer-record/'>)\n           \u2514 <function ClientRecordView.post at 0x7f52106f1d30>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py\", line 52, in post\n    client = Client.objects.get(user=request.user)\n             \u2502      \u2502                \u2502       \u2514 <SimpleLazyObject: <User: admin>>\n             \u2502      \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n             \u2502      \u2514 <django.db.models.manager.ManagerDescriptor object at 0x7f52123f4bb0>\n             \u2514 <class 'email_send.models.Client'>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/manager.py\", line 85, in manager_method\n    return getattr(self.get_queryset(), name)(*args, **kwargs)\n                   \u2502    \u2502               \u2502      \u2502       \u2514 {'user': <SimpleLazyObject: <User: admin>>}\n                   \u2502    \u2502               \u2502      \u2514 ()\n                   \u2502    \u2502               \u2514 'get'\n                   \u2502    \u2514 <function BaseManager.get_queryset at 0x7f521507c550>\n                   \u2514 <django.db.models.manager.Manager object at 0x7f52106b3490>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/query.py\", line 429, in get\n    raise self.model.DoesNotExist(\n          \u2502    \u2502     \u2514 <class 'email_send.models.Client.DoesNotExist'>\n          \u2502    \u2514 <class 'email_send.models.Client'>\n          \u2514 <QuerySet []>\n\nemail_send.models.Client.DoesNotExist: Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:16.993221", "seconds": 16.993221}, "exception": {"type": "DoesNotExist", "value": "Client matching query does not exist.", "traceback": true}, "extra": {}, "file": {"name": "base.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py"}, "function": "dispatch", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 98, "message": "An error has been caught in function 'dispatch', process 'MainProcess' (275764), thread 'Thread-5' (139990356977408):", "module": "base", "name": "django.views.generic.base", "process": {"id": 275764, "name": "MainProcess"}, "thread": {"id": 139990356977408, "name": "Thread-5"}, "time": {"repr": "2021-05-08 15:40:46.989782+00:00", "timestamp": 1620488446.989782}}}
{"text": "2021-05-08T15:42:35.315279+0000 ERROR An error has been caught in function 'dispatch', process 'MainProcess' (276728), thread 'Thread-1' (139913475499776):\nTraceback (most recent call last):\n\n  File \"/usr/lib/python3.8/threading.py\", line 890, in _bootstrap\n    self._bootstrap_inner()\n    \u2502    \u2514 <function Thread._bootstrap_inner at 0x7f402aecb4c0>\n    \u2514 <Thread(Thread-1, started daemon 139913475499776)>\n  File \"/usr/lib/python3.8/threading.py\", line 932, in _bootstrap_inner\n    self.run()\n    \u2502    \u2514 <function Thread.run at 0x7f402aecb1f0>\n    \u2514 <Thread(Thread-1, started daemon 139913475499776)>\n  File \"/usr/lib/python3.8/threading.py\", line 870, in run\n    self._target(*self._args, **self._kwargs)\n    \u2502    \u2502        \u2502    \u2502        \u2502    \u2514 {}\n    \u2502    \u2502        \u2502    \u2502        \u2514 <Thread(Thread-1, started daemon 139913475499776)>\n    \u2502    \u2502        \u2502    \u2514 (<socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('...\n    \u2502    \u2502        \u2514 <Thread(Thread-1, started daemon 139913475499776)>\n    \u2502    \u2514 <bound method ThreadingMixIn.process_request_thread of <django.core.servers.basehttp.WSGIServer object at 0x7f40251428e0>>\n    \u2514 <Thread(Thread-1, started daemon 139913475499776)>\n  File \"/usr/lib/python3.8/socketserver.py\", line 650, in process_request_thread\n    self.finish_request(request, client_address)\n    \u2502    \u2502              \u2502        \u2514 ('127.0.0.1', 49076)\n    \u2502    \u2502              \u2514 <socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <function BaseServer.finish_request at 0x7f4029c03e50>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f40251428e0>\n  File \"/usr/lib/python3.8/socketserver.py\", line 360, in finish_request\n    self.RequestHandlerClass(request, client_address, self)\n    \u2502    \u2502                   \u2502        \u2502               \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f40251428e0>\n    \u2502    \u2502                   \u2502        \u2514 ('127.0.0.1', 49076)\n    \u2502    \u2502                   \u2514 <socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <class 'django.core.servers.basehttp.WSGIRequestHandler'>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f40251428e0>\n  File \"/usr/lib/python3.8/socketserver.py\", line 720, in __init__\n    self.handle()\n    \u2502    \u2514 <function WSGIRequestHandler.handle at 0x7f4026e7de50>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f402505bdf0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 172, in handle\n    self.handle_one_request()\n    \u2502    \u2514 <function WSGIRequestHandler.handle_one_request at 0x7f4026e7dee0>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f402505bdf0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 197, in handle_one_request\n    handler.run(self.server.get_app())\n    \u2502       \u2502   \u2502    \u2502      \u2514 <function WSGIServer.get_app at 0x7f4026e7d430>\n    \u2502       \u2502   \u2502    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f40251428e0>\n    \u2502       \u2502   \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f402505bdf0>\n    \u2502       \u2514 <function BaseHandler.run at 0x7f4026ef70d0>\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f402516fe80>\n  File \"/usr/lib/python3.8/wsgiref/handlers.py\", line 137, in run\n    self.result = application(self.environ, self.start_response)\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2502    \u2514 <function BaseHandler.start_response at 0x7f4026ef7430>\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f402516fe80>\n    \u2502    \u2502        \u2502           \u2502    \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n    \u2502    \u2502        \u2502           \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f402516fe80>\n    \u2502    \u2502        \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f40251429d0>\n    \u2502    \u2514 None\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f402516fe80>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/contrib/staticfiles/handlers.py\", line 76, in __call__\n    return self.application(environ, start_response)\n           \u2502    \u2502           \u2502        \u2514 <bound method BaseHandler.start_response of <django.core.servers.basehttp.ServerHandler object at 0x7f402516fe80>>\n           \u2502    \u2502           \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n           \u2502    \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f402a5ebbe0>\n           \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f40251429d0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/wsgi.py\", line 133, in __call__\n    response = self.get_response(request)\n               \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function BaseHandler.get_response at 0x7f4026f21f70>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f402a5ebbe0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 130, in get_response\n    response = self._middleware_chain(request)\n               \u2502    \u2502                 \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f4029a459d0>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f402a5ebbe0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f402a5ddd90>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f4029a45a60>\n               \u2502           \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f402a5ddd90>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f4029a48070>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f4029a45af0>\n               \u2502           \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f4029a48070>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.common.CommonMiddleware object at 0x7f4029a480d0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f4029a45b80>\n               \u2502           \u2514 <django.middleware.common.CommonMiddleware object at 0x7f4029a480d0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f4029a48100>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f4029a45c10>\n               \u2502           \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f4029a48100>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f4029a48310>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f4029a45dc0>\n               \u2502           \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f4029a48310>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f4029a482e0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f4029a45e50>\n               \u2502           \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f4029a482e0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f4029a483a0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function BaseHandler._get_response at 0x7f4029a45550>\n               \u2502           \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f4029a483a0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <bound method BaseHandler._get_response of <django.core.handlers.wsgi.WSGIHandler object at 0x7f402a5ebbe0>>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 181, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n               \u2502                \u2502         \u2502                \u2514 {}\n               \u2502                \u2502         \u2514 ()\n               \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <function ClientRecordView at 0x7f40265ee280>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 70, in view\n    return self.dispatch(request, *args, **kwargs)\n           \u2502    \u2502        \u2502         \u2502       \u2514 {}\n           \u2502    \u2502        \u2502         \u2514 ()\n           \u2502    \u2502        \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2502    \u2514 <function View.dispatch at 0x7f4029a56a60>\n           \u2514 <email_send.views.ClientRecordView object at 0x7f4025142e80>\n> File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 98, in dispatch\n    return handler(request, *args, **kwargs)\n           \u2502       \u2502         \u2502       \u2514 {}\n           \u2502       \u2502         \u2514 ()\n           \u2502       \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2514 <bound method ClientRecordView.post of <email_send.views.ClientRecordView object at 0x7f4025142e80>>\n  File \"/usr/lib/python3.8/contextlib.py\", line 75, in inner\n    return func(*args, **kwds)\n           \u2502     \u2502       \u2514 {}\n           \u2502     \u2514 (<email_send.views.ClientRecordView object at 0x7f4025142e80>, <WSGIRequest: POST '/contact/customer-record/'>)\n           \u2514 <function ClientRecordView.post at 0x7f40252e3d30>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py\", line 38, in post\n    client = Client.objects.get(user=request.user)\n             \u2502      \u2502                \u2502       \u2514 <SimpleLazyObject: <User: admin>>\n             \u2502      \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n             \u2502      \u2514 <django.db.models.manager.ManagerDescriptor object at 0x7f4026fe6bb0>\n             \u2514 <class 'email_send.models.Client'>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/manager.py\", line 85, in manager_method\n    return getattr(self.get_queryset(), name)(*args, **kwargs)\n                   \u2502    \u2502               \u2502      \u2502       \u2514 {'user': <SimpleLazyObject: <User: admin>>}\n                   \u2502    \u2502               \u2502      \u2514 ()\n                   \u2502    \u2502               \u2514 'get'\n                   \u2502    \u2514 <function BaseManager.get_queryset at 0x7f4029c6e550>\n                   \u2514 <django.db.models.manager.Manager object at 0x7f40252a5580>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/query.py\", line 429, in get\n    raise self.model.DoesNotExist(\n          \u2502    \u2502     \u2514 <class 'email_send.models.Client.DoesNotExist'>\n          \u2502    \u2514 <class 'email_send.models.Client'>\n          \u2514 <QuerySet []>\n\nemail_send.models.Client.DoesNotExist: Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:36.609581", "seconds": 36.609581}, "exception": {"type": "DoesNotExist", "value": "Client matching query does not exist.", "traceback": true}, "extra": {}, "file": {"name": "base.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py"}, "function": "dispatch", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 98, "message": "An error has been caught in function 'dispatch', process 'MainProcess' (276728), thread 'Thread-1' (139913475499776):", "module": "base", "name": "django.views.generic.base", "process": {"id": 276728, "name": "MainProcess"}, "thread": {"id": 139913475499776, "name": "Thread-1"}, "time": {"repr": "2021-05-08 15:42:35.315279+00:00", "timestamp": 1620488555.315279}}}
{"text": "2021-05-08T15:51:08.513830+0000 ERROR An error has been caught in function 'dispatch', process 'MainProcess' (280951), thread 'Thread-1' (139818267301632):\nTraceback (most recent call last):\n\n  File \"/usr/lib/python3.8/threading.py\", line 890, in _bootstrap\n    self._bootstrap_inner()\n    \u2502    \u2514 <function Thread._bootstrap_inner at 0x7f2a000ae4c0>\n    \u2514 <Thread(Thread-1, started daemon 139818267301632)>\n  File \"/usr/lib/python3.8/threading.py\", line 932, in _bootstrap_inner\n    self.run()\n    \u2502    \u2514 <function Thread.run at 0x7f2a000ae1f0>\n    \u2514 <Thread(Thread-1, started daemon 139818267301632)>\n  File \"/usr/lib/python3.8/threading.py\", line 870, in run\n    self._target(*self._args, **self._kwargs)\n    \u2502    \u2502        \u2502    \u2502        \u2502    \u2514 {}\n    \u2502    \u2502        \u2502    \u2502        \u2514 <Thread(Thread-1, started daemon 139818267301632)>\n    \u2502    \u2502        \u2502    \u2514 (<socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('...\n    \u2502    \u2502        \u2514 <Thread(Thread-1, started daemon 139818267301632)>\n    \u2502    \u2514 <bound method ThreadingMixIn.process_request_thread of <django.core.servers.basehttp.WSGIServer object at 0x7f29fa361370>>\n    \u2514 <Thread(Thread-1, started daemon 139818267301632)>\n  File \"/usr/lib/python3.8/socketserver.py\", line 650, in process_request_thread\n    self.finish_request(request, client_address)\n    \u2502    \u2502              \u2502        \u2514 ('127.0.0.1', 49256)\n    \u2502    \u2502              \u2514 <socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <function BaseServer.finish_request at 0x7f29fede6e50>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f29fa361370>\n  File \"/usr/lib/python3.8/socketserver.py\", line 360, in finish_request\n    self.RequestHandlerClass(request, client_address, self)\n    \u2502    \u2502                   \u2502        \u2502               \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f29fa361370>\n    \u2502    \u2502                   \u2502        \u2514 ('127.0.0.1', 49256)\n    \u2502    \u2502                   \u2514 <socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <class 'django.core.servers.basehttp.WSGIRequestHandler'>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f29fa361370>\n  File \"/usr/lib/python3.8/socketserver.py\", line 720, in __init__\n    self.handle()\n    \u2502    \u2514 <function WSGIRequestHandler.handle at 0x7f29fc060e50>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f29fa2fa640>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 174, in handle\n    self.handle_one_request()\n    \u2502    \u2514 <function WSGIRequestHandler.handle_one_request at 0x7f29fc060ee0>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f29fa2fa640>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 197, in handle_one_request\n    handler.run(self.server.get_app())\n    \u2502       \u2502   \u2502    \u2502      \u2514 <function WSGIServer.get_app at 0x7f29fc060430>\n    \u2502       \u2502   \u2502    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f29fa361370>\n    \u2502       \u2502   \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f29fa2fa640>\n    \u2502       \u2514 <function BaseHandler.run at 0x7f29fc0da0d0>\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f29f99c2e20>\n  File \"/usr/lib/python3.8/wsgiref/handlers.py\", line 137, in run\n    self.result = application(self.environ, self.start_response)\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2502    \u2514 <function BaseHandler.start_response at 0x7f29fc0da430>\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f29f99c2e20>\n    \u2502    \u2502        \u2502           \u2502    \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n    \u2502    \u2502        \u2502           \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f29f99c2e20>\n    \u2502    \u2502        \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f29fc06f130>\n    \u2502    \u2514 None\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f29f99c2e20>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/contrib/staticfiles/handlers.py\", line 76, in __call__\n    return self.application(environ, start_response)\n           \u2502    \u2502           \u2502        \u2514 <bound method BaseHandler.start_response of <django.core.servers.basehttp.ServerHandler object at 0x7f29f99c2e20>>\n           \u2502    \u2502           \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n           \u2502    \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f29ff7cebe0>\n           \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f29fc06f130>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/wsgi.py\", line 133, in __call__\n    response = self.get_response(request)\n               \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function BaseHandler.get_response at 0x7f29fc104f70>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f29ff7cebe0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 130, in get_response\n    response = self._middleware_chain(request)\n               \u2502    \u2502                 \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f29fec289d0>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f29ff7cebe0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f29ff7c0d90>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f29fec28a60>\n               \u2502           \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f29ff7c0d90>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f29fec2b070>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f29fec28af0>\n               \u2502           \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f29fec2b070>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.common.CommonMiddleware object at 0x7f29fec2b0d0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f29fec28b80>\n               \u2502           \u2514 <django.middleware.common.CommonMiddleware object at 0x7f29fec2b0d0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f29fec2b100>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f29fec28c10>\n               \u2502           \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f29fec2b100>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f29fec2b310>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f29fec28dc0>\n               \u2502           \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f29fec2b310>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f29fec2b2e0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f29fec28e50>\n               \u2502           \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f29fec2b2e0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f29fec2b3a0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function BaseHandler._get_response at 0x7f29fec28550>\n               \u2502           \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f29fec2b3a0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <bound method BaseHandler._get_response of <django.core.handlers.wsgi.WSGIHandler object at 0x7f29ff7cebe0>>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 181, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n               \u2502                \u2502         \u2502                \u2514 {}\n               \u2502                \u2502         \u2514 ()\n               \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <function ClientRecordView at 0x7f29fb7d1280>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 70, in view\n    return self.dispatch(request, *args, **kwargs)\n           \u2502    \u2502        \u2502         \u2502       \u2514 {}\n           \u2502    \u2502        \u2502         \u2514 ()\n           \u2502    \u2502        \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2502    \u2514 <function View.dispatch at 0x7f29fec38a60>\n           \u2514 <email_send.views.ClientRecordView object at 0x7f29f9a57940>\n> File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 98, in dispatch\n    return handler(request, *args, **kwargs)\n           \u2502       \u2502         \u2502       \u2514 {}\n           \u2502       \u2502         \u2514 ()\n           \u2502       \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2514 <bound method ClientRecordView.post of <email_send.views.ClientRecordView object at 0x7f29f9a57940>>\n  File \"/usr/lib/python3.8/contextlib.py\", line 75, in inner\n    return func(*args, **kwds)\n           \u2502     \u2502       \u2514 {}\n           \u2502     \u2514 (<email_send.views.ClientRecordView object at 0x7f29f9a57940>, <WSGIRequest: POST '/contact/customer-record/'>)\n           \u2514 <function ClientRecordView.post at 0x7f29fa4c6d30>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py\", line 40, in post\n    client = Client.objects.get(user=request.user)\n             \u2502      \u2502                \u2502       \u2514 <SimpleLazyObject: <User: qwe>>\n             \u2502      \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n             \u2502      \u2514 <django.db.models.manager.ManagerDescriptor object at 0x7f29fc1c9bb0>\n             \u2514 <class 'email_send.models.Client'>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/manager.py\", line 85, in manager_method\n    return getattr(self.get_queryset(), name)(*args, **kwargs)\n                   \u2502    \u2502               \u2502      \u2502       \u2514 {'user': <SimpleLazyObject: <User: qwe>>}\n                   \u2502    \u2502               \u2502      \u2514 ()\n                   \u2502    \u2502               \u2514 'get'\n                   \u2502    \u2514 <function BaseManager.get_queryset at 0x7f29fee51550>\n                   \u2514 <django.db.models.manager.Manager object at 0x7f29fa48a460>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/query.py\", line 429, in get\n    raise self.model.DoesNotExist(\n          \u2502    \u2502     \u2514 <class 'email_send.models.Client.DoesNotExist'>\n          \u2502    \u2514 <class 'email_send.models.Client'>\n          \u2514 <QuerySet []>\n\nemail_send.models.Client.DoesNotExist: Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:38.714959", "seconds": 38.714959}, "exception": {"type": "DoesNotExist", "value": "Client matching query does not exist.", "traceback": true}, "extra": {}, "file": {"name": "base.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py"}, "function": "dispatch", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 98, "message": "An error has been caught in function 'dispatch', process 'MainProcess' (280951), thread 'Thread-1' (139818267301632):", "module": "base", "name": "django.views.generic.base", "process": {"id": 280951, "name": "MainProcess"}, "thread": {"id": 139818267301632, "name": "Thread-1"}, "time": {"repr": "2021-05-08 15:51:08.513830+00:00", "timestamp": 1620489068.51383}}}
{"text": "2021-05-08T16:24:54.494455+0000 ERROR Error Client matching query does not exist.____not client\n", "record": {"elapsed": {"repr": "0:04:44.011950", "seconds": 284.01195}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 54, "message": "Error Client matching query does not exist.____not client", "module": "views", "name": "email_send.views", "process": {"id": 291714, "name": "MainProcess"}, "thread": {"id": 140429928429312, "name": "Thread-6"}, "time": {"repr": "2021-05-08 16:24:54.494455+00:00", "timestamp": 1620491094.494455}}}
{"text": "2021-05-08T16:27:05.104012+0000 ERROR An error has been caught in function 'dispatch', process 'MainProcess' (294601), thread 'Thread-1' (140021801465600):\nTraceback (most recent call last):\n\n  File \"/usr/lib/python3.8/threading.py\", line 890, in _bootstrap\n    self._bootstrap_inner()\n    \u2502    \u2514 <function Thread._bootstrap_inner at 0x7f5963a7c4c0>\n    \u2514 <Thread(Thread-1, started daemon 140021801465600)>\n  File \"/usr/lib/python3.8/threading.py\", line 932, in _bootstrap_inner\n    self.run()\n    \u2502    \u2514 <function Thread.run at 0x7f5963a7c1f0>\n    \u2514 <Thread(Thread-1, started daemon 140021801465600)>\n  File \"/usr/lib/python3.8/threading.py\", line 870, in run\n    self._target(*self._args, **self._kwargs)\n    \u2502    \u2502        \u2502    \u2502        \u2502    \u2514 {}\n    \u2502    \u2502        \u2502    \u2502        \u2514 <Thread(Thread-1, started daemon 140021801465600)>\n    \u2502    \u2502        \u2502    \u2514 (<socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('...\n    \u2502    \u2502        \u2514 <Thread(Thread-1, started daemon 140021801465600)>\n    \u2502    \u2514 <bound method ThreadingMixIn.process_request_thread of <django.core.servers.basehttp.WSGIServer object at 0x7f595dd4e520>>\n    \u2514 <Thread(Thread-1, started daemon 140021801465600)>\n  File \"/usr/lib/python3.8/socketserver.py\", line 650, in process_request_thread\n    self.finish_request(request, client_address)\n    \u2502    \u2502              \u2502        \u2514 ('127.0.0.1', 49752)\n    \u2502    \u2502              \u2514 <socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <function BaseServer.finish_request at 0x7f59627b4e50>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f595dd4e520>\n  File \"/usr/lib/python3.8/socketserver.py\", line 360, in finish_request\n    self.RequestHandlerClass(request, client_address, self)\n    \u2502    \u2502                   \u2502        \u2502               \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f595dd4e520>\n    \u2502    \u2502                   \u2502        \u2514 ('127.0.0.1', 49752)\n    \u2502    \u2502                   \u2514 <socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <class 'django.core.servers.basehttp.WSGIRequestHandler'>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f595dd4e520>\n  File \"/usr/lib/python3.8/socketserver.py\", line 720, in __init__\n    self.handle()\n    \u2502    \u2514 <function WSGIRequestHandler.handle at 0x7f595fa2ee50>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f595dc29730>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 174, in handle\n    self.handle_one_request()\n    \u2502    \u2514 <function WSGIRequestHandler.handle_one_request at 0x7f595fa2eee0>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f595dc29730>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 197, in handle_one_request\n    handler.run(self.server.get_app())\n    \u2502       \u2502   \u2502    \u2502      \u2514 <function WSGIServer.get_app at 0x7f595fa2e430>\n    \u2502       \u2502   \u2502    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f595dd4e520>\n    \u2502       \u2502   \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f595dc29730>\n    \u2502       \u2514 <function BaseHandler.run at 0x7f595faa80d0>\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f595ddee490>\n  File \"/usr/lib/python3.8/wsgiref/handlers.py\", line 137, in run\n    self.result = application(self.environ, self.start_response)\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2502    \u2514 <function BaseHandler.start_response at 0x7f595faa8430>\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f595ddee490>\n    \u2502    \u2502        \u2502           \u2502    \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n    \u2502    \u2502        \u2502           \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f595ddee490>\n    \u2502    \u2502        \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f595fa3d3a0>\n    \u2502    \u2514 None\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f595ddee490>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/contrib/staticfiles/handlers.py\", line 76, in __call__\n    return self.application(environ, start_response)\n           \u2502    \u2502           \u2502        \u2514 <bound method BaseHandler.start_response of <django.core.servers.basehttp.ServerHandler object at 0x7f595ddee490>>\n           \u2502    \u2502           \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n           \u2502    \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f596319cbe0>\n           \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f595fa3d3a0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/wsgi.py\", line 133, in __call__\n    response = self.get_response(request)\n               \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function BaseHandler.get_response at 0x7f595fad2f70>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f596319cbe0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 130, in get_response\n    response = self._middleware_chain(request)\n               \u2502    \u2502                 \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f59625f65e0>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f596319cbe0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f596318edf0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f59625f68b0>\n               \u2502           \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f596318edf0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f596318ee20>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f59625f6940>\n               \u2502           \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f596318ee20>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.common.CommonMiddleware object at 0x7f596318ef10>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f59625f69d0>\n               \u2502           \u2514 <django.middleware.common.CommonMiddleware object at 0x7f596318ef10>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f596318ef40>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f59625f6a60>\n               \u2502           \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f596318ef40>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f596318ed90>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f59625f6c10>\n               \u2502           \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f596318ed90>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f59625f92e0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f59625f6e50>\n               \u2502           \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f59625f92e0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f59625f93a0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function BaseHandler._get_response at 0x7f59625f6550>\n               \u2502           \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f59625f93a0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <bound method BaseHandler._get_response of <django.core.handlers.wsgi.WSGIHandler object at 0x7f596319cbe0>>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 181, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n               \u2502                \u2502         \u2502                \u2514 {}\n               \u2502                \u2502         \u2514 ()\n               \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <function ClientRecordView at 0x7f595f19f280>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 70, in view\n    return self.dispatch(request, *args, **kwargs)\n           \u2502    \u2502        \u2502         \u2502       \u2514 {}\n           \u2502    \u2502        \u2502         \u2514 ()\n           \u2502    \u2502        \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2502    \u2514 <function View.dispatch at 0x7f5962607a60>\n           \u2514 <email_send.views.ClientRecordView object at 0x7f595dd3ddc0>\n> File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 98, in dispatch\n    return handler(request, *args, **kwargs)\n           \u2502       \u2502         \u2502       \u2514 {}\n           \u2502       \u2502         \u2514 ()\n           \u2502       \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2514 <bound method ClientRecordView.post of <email_send.views.ClientRecordView object at 0x7f595dd3ddc0>>\n  File \"/usr/lib/python3.8/contextlib.py\", line 75, in inner\n    return func(*args, **kwds)\n           \u2502     \u2502       \u2514 {}\n           \u2502     \u2514 (<email_send.views.ClientRecordView object at 0x7f595dd3ddc0>, <WSGIRequest: POST '/contact/customer-record/'>)\n           \u2514 <function ClientRecordView.post at 0x7f595de95d30>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py\", line 41, in post\n    client = Client.objects.get(user=request.user)\n             \u2502      \u2502                \u2502       \u2514 <SimpleLazyObject: <User: admin>>\n             \u2502      \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n             \u2502      \u2514 <django.db.models.manager.ManagerDescriptor object at 0x7f595fb97bb0>\n             \u2514 <class 'email_send.models.Client'>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/manager.py\", line 85, in manager_method\n    return getattr(self.get_queryset(), name)(*args, **kwargs)\n                   \u2502    \u2502               \u2502      \u2502       \u2514 {'user': <SimpleLazyObject: <User: admin>>}\n                   \u2502    \u2502               \u2502      \u2514 ()\n                   \u2502    \u2502               \u2514 'get'\n                   \u2502    \u2514 <function BaseManager.get_queryset at 0x7f596281f550>\n                   \u2514 <django.db.models.manager.Manager object at 0x7f595de57430>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/query.py\", line 429, in get\n    raise self.model.DoesNotExist(\n          \u2502    \u2502     \u2514 <class 'email_send.models.Client.DoesNotExist'>\n          \u2502    \u2514 <class 'email_send.models.Client'>\n          \u2514 <QuerySet [<Client: \u041a\u043b\u0438\u0435\u043d\u0442 danil danil>]>\n\nemail_send.models.Client.DoesNotExist: Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:11.407565", "seconds": 11.407565}, "exception": {"type": "DoesNotExist", "value": "Client matching query does not exist.", "traceback": true}, "extra": {}, "file": {"name": "base.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py"}, "function": "dispatch", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 98, "message": "An error has been caught in function 'dispatch', process 'MainProcess' (294601), thread 'Thread-1' (140021801465600):", "module": "base", "name": "django.views.generic.base", "process": {"id": 294601, "name": "MainProcess"}, "thread": {"id": 140021801465600, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:27:05.104012+00:00", "timestamp": 1620491225.104012}}}
{"text": "2021-05-08T16:31:38.286571+0000 ERROR Error Client matching query does not exist.____not client\n", "record": {"elapsed": {"repr": "0:01:35.854144", "seconds": 95.854144}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 56, "message": "Error Client matching query does not exist.____not client", "module": "views", "name": "email_send.views", "process": {"id": 296265, "name": "MainProcess"}, "thread": {"id": 139670267315968, "name": "Thread-2"}, "time": {"repr": "2021-05-08 16:31:38.286571+00:00", "timestamp": 1620491498.286571}}}
{"text": "2021-05-08T16:31:38.287417+0000 ERROR An error has been caught in function 'dispatch', process 'MainProcess' (296265), thread 'Thread-2' (139670267315968):\nTraceback (most recent call last):\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py\", line 41, in post\n    client = Client.objects.get(user=request.user)\n             \u2502      \u2502                \u2502       \u2514 <SimpleLazyObject: <User: admin>>\n             \u2502      \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n             \u2502      \u2514 <django.db.models.manager.ManagerDescriptor object at 0x7f0787781bb0>\n             \u2514 <class 'email_send.models.Client'>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/manager.py\", line 85, in manager_method\n    return getattr(self.get_queryset(), name)(*args, **kwargs)\n                   \u2502    \u2502               \u2502      \u2502       \u2514 {'user': <SimpleLazyObject: <User: admin>>}\n                   \u2502    \u2502               \u2502      \u2514 ()\n                   \u2502    \u2502               \u2514 'get'\n                   \u2502    \u2514 <function BaseManager.get_queryset at 0x7f078a409550>\n                   \u2514 <django.db.models.manager.Manager object at 0x7f0785a415b0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/query.py\", line 429, in get\n    raise self.model.DoesNotExist(\n          \u2502    \u2502     \u2514 <class 'email_send.models.Client.DoesNotExist'>\n          \u2502    \u2514 <class 'email_send.models.Client'>\n          \u2514 <QuerySet [<Client: \u041a\u043b\u0438\u0435\u043d\u0442 danil danil>]>\n\nemail_send.models.Client.DoesNotExist: Client matching query does not exist.\n\n\nDuring handling of the above exception, another exception occurred:\n\n\nTraceback (most recent call last):\n\n  File \"/usr/lib/python3.8/threading.py\", line 890, in _bootstrap\n    self._bootstrap_inner()\n    \u2502    \u2514 <function Thread._bootstrap_inner at 0x7f078b6664c0>\n    \u2514 <Thread(Thread-2, started daemon 139670267315968)>\n  File \"/usr/lib/python3.8/threading.py\", line 932, in _bootstrap_inner\n    self.run()\n    \u2502    \u2514 <function Thread.run at 0x7f078b6661f0>\n    \u2514 <Thread(Thread-2, started daemon 139670267315968)>\n  File \"/usr/lib/python3.8/threading.py\", line 870, in run\n    self._target(*self._args, **self._kwargs)\n    \u2502    \u2502        \u2502    \u2502        \u2502    \u2514 {}\n    \u2502    \u2502        \u2502    \u2502        \u2514 <Thread(Thread-2, started daemon 139670267315968)>\n    \u2502    \u2502        \u2502    \u2514 (<socket.socket fd=9, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('...\n    \u2502    \u2502        \u2514 <Thread(Thread-2, started daemon 139670267315968)>\n    \u2502    \u2514 <bound method ThreadingMixIn.process_request_thread of <django.core.servers.basehttp.WSGIServer object at 0x7f078592e070>>\n    \u2514 <Thread(Thread-2, started daemon 139670267315968)>\n  File \"/usr/lib/python3.8/socketserver.py\", line 650, in process_request_thread\n    self.finish_request(request, client_address)\n    \u2502    \u2502              \u2502        \u2514 ('127.0.0.1', 49838)\n    \u2502    \u2502              \u2514 <socket.socket fd=9, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <function BaseServer.finish_request at 0x7f078a39ee50>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f078592e070>\n  File \"/usr/lib/python3.8/socketserver.py\", line 360, in finish_request\n    self.RequestHandlerClass(request, client_address, self)\n    \u2502    \u2502                   \u2502        \u2502               \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f078592e070>\n    \u2502    \u2502                   \u2502        \u2514 ('127.0.0.1', 49838)\n    \u2502    \u2502                   \u2514 <socket.socket fd=9, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <class 'django.core.servers.basehttp.WSGIRequestHandler'>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f078592e070>\n  File \"/usr/lib/python3.8/socketserver.py\", line 720, in __init__\n    self.handle()\n    \u2502    \u2514 <function WSGIRequestHandler.handle at 0x7f0787618e50>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f0784f261c0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 174, in handle\n    self.handle_one_request()\n    \u2502    \u2514 <function WSGIRequestHandler.handle_one_request at 0x7f0787618ee0>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f0784f261c0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 197, in handle_one_request\n    handler.run(self.server.get_app())\n    \u2502       \u2502   \u2502    \u2502      \u2514 <function WSGIServer.get_app at 0x7f0787618430>\n    \u2502       \u2502   \u2502    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f078592e070>\n    \u2502       \u2502   \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f0784f261c0>\n    \u2502       \u2514 <function BaseHandler.run at 0x7f07876920d0>\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f07842884c0>\n  File \"/usr/lib/python3.8/wsgiref/handlers.py\", line 137, in run\n    self.result = application(self.environ, self.start_response)\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2502    \u2514 <function BaseHandler.start_response at 0x7f0787692430>\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f07842884c0>\n    \u2502    \u2502        \u2502           \u2502    \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n    \u2502    \u2502        \u2502           \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f07842884c0>\n    \u2502    \u2502        \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f0786e0d3d0>\n    \u2502    \u2514 None\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f07842884c0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/contrib/staticfiles/handlers.py\", line 76, in __call__\n    return self.application(environ, start_response)\n           \u2502    \u2502           \u2502        \u2514 <bound method BaseHandler.start_response of <django.core.servers.basehttp.ServerHandler object at 0x7f07842884c0>>\n           \u2502    \u2502           \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n           \u2502    \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f078ad86be0>\n           \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f0786e0d3d0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/wsgi.py\", line 133, in __call__\n    response = self.get_response(request)\n               \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function BaseHandler.get_response at 0x7f07876bcf70>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f078ad86be0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 130, in get_response\n    response = self._middleware_chain(request)\n               \u2502    \u2502                 \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f07859801f0>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f078ad86be0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f078593b7f0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f078587a3a0>\n               \u2502           \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f078593b7f0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f078593b040>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f078587a310>\n               \u2502           \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f078593b040>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.common.CommonMiddleware object at 0x7f07858e5d00>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f078a1e05e0>\n               \u2502           \u2514 <django.middleware.common.CommonMiddleware object at 0x7f07858e5d00>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f07858e55e0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f078a1e0670>\n               \u2502           \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f07858e55e0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f07858e5490>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f078a1e0820>\n               \u2502           \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f07858e5490>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f078a1e32e0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f078a1e0e50>\n               \u2502           \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f078a1e32e0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f078a1e33a0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function BaseHandler._get_response at 0x7f078a1e0550>\n               \u2502           \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f078a1e33a0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <bound method BaseHandler._get_response of <django.core.handlers.wsgi.WSGIHandler object at 0x7f078ad86be0>>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 181, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n               \u2502                \u2502         \u2502                \u2514 {}\n               \u2502                \u2502         \u2514 ()\n               \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <function ClientRecordView at 0x7f0786d89280>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 70, in view\n    return self.dispatch(request, *args, **kwargs)\n           \u2502    \u2502        \u2502         \u2502       \u2514 {}\n           \u2502    \u2502        \u2502         \u2514 ()\n           \u2502    \u2502        \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2502    \u2514 <function View.dispatch at 0x7f078a1f0a60>\n           \u2514 <email_send.views.ClientRecordView object at 0x7f0784ebc640>\n> File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 98, in dispatch\n    return handler(request, *args, **kwargs)\n           \u2502       \u2502         \u2502       \u2514 {}\n           \u2502       \u2502         \u2514 ()\n           \u2502       \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2514 <bound method ClientRecordView.post of <email_send.views.ClientRecordView object at 0x7f0784ebc640>>\n  File \"/usr/lib/python3.8/contextlib.py\", line 75, in inner\n    return func(*args, **kwds)\n           \u2502     \u2502       \u2514 {}\n           \u2502     \u2514 (<email_send.views.ClientRecordView object at 0x7f0784ebc640>, <WSGIRequest: POST '/contact/customer-record/'>)\n           \u2514 <function ClientRecordView.post at 0x7f0785a7fdc0>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py\", line 59, in post\n    phone=form.cleaned_data['phone'],\n          \u2514 <ClientRecordForm bound=True, valid=Unknown, fields=(email;first_name;last_name;phone;message)>\n\nAttributeError: 'ClientRecordForm' object has no attribute 'cleaned_data'\n", "record": {"elapsed": {"repr": "0:01:35.854990", "seconds": 95.85499}, "exception": {"type": "AttributeError", "value": "'ClientRecordForm' object has no attribute 'cleaned_data'", "traceback": true}, "extra": {}, "file": {"name": "base.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py"}, "function": "dispatch", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 98, "message": "An error has been caught in function 'dispatch', process 'MainProcess' (296265), thread 'Thread-2' (139670267315968):", "module": "base", "name": "django.views.generic.base", "process": {"id": 296265, "name": "MainProcess"}, "thread": {"id": 139670267315968, "name": "Thread-2"}, "time": {"repr": "2021-05-08 16:31:38.287417+00:00", "timestamp": 1620491498.287417}}}
{"text": "2021-05-08T16:36:57.032322+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:27.348446", "seconds": 27.348446}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 55, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 299448, "name": "MainProcess"}, "thread": {"id": 139873897260800, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:36:57.032322+00:00", "timestamp": 1620491817.032322}}}
{"text": "2021-05-08T16:36:57.033549+0000 ERROR An error has been caught in function 'dispatch', process 'MainProcess' (299448), thread 'Thread-1' (139873897260800):\nTraceback (most recent call last):\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py\", line 41, in post\n    client = Client.objects.get(user=request.user)\n             \u2502      \u2502                \u2502       \u2514 <SimpleLazyObject: <User: admin>>\n             \u2502      \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n             \u2502      \u2514 <django.db.models.manager.ManagerDescriptor object at 0x7f36eff26bb0>\n             \u2514 <class 'email_send.models.Client'>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/manager.py\", line 85, in manager_method\n    return getattr(self.get_queryset(), name)(*args, **kwargs)\n                   \u2502    \u2502               \u2502      \u2502       \u2514 {'user': <SimpleLazyObject: <User: admin>>}\n                   \u2502    \u2502               \u2502      \u2514 ()\n                   \u2502    \u2502               \u2514 'get'\n                   \u2502    \u2514 <function BaseManager.get_queryset at 0x7f36f2bae550>\n                   \u2514 <django.db.models.manager.Manager object at 0x7f36ee1e53d0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/query.py\", line 429, in get\n    raise self.model.DoesNotExist(\n          \u2502    \u2502     \u2514 <class 'email_send.models.Client.DoesNotExist'>\n          \u2502    \u2514 <class 'email_send.models.Client'>\n          \u2514 <QuerySet [<Client: \u041a\u043b\u0438\u0435\u043d\u0442 danil danil>]>\n\nemail_send.models.Client.DoesNotExist: Client matching query does not exist.\n\n\nDuring handling of the above exception, another exception occurred:\n\n\nTraceback (most recent call last):\n\n  File \"/usr/lib/python3.8/threading.py\", line 890, in _bootstrap\n    self._bootstrap_inner()\n    \u2502    \u2514 <function Thread._bootstrap_inner at 0x7f36f3e0b4c0>\n    \u2514 <Thread(Thread-1, started daemon 139873897260800)>\n  File \"/usr/lib/python3.8/threading.py\", line 932, in _bootstrap_inner\n    self.run()\n    \u2502    \u2514 <function Thread.run at 0x7f36f3e0b1f0>\n    \u2514 <Thread(Thread-1, started daemon 139873897260800)>\n  File \"/usr/lib/python3.8/threading.py\", line 870, in run\n    self._target(*self._args, **self._kwargs)\n    \u2502    \u2502        \u2502    \u2502        \u2502    \u2514 {}\n    \u2502    \u2502        \u2502    \u2502        \u2514 <Thread(Thread-1, started daemon 139873897260800)>\n    \u2502    \u2502        \u2502    \u2514 (<socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('...\n    \u2502    \u2502        \u2514 <Thread(Thread-1, started daemon 139873897260800)>\n    \u2502    \u2514 <bound method ThreadingMixIn.process_request_thread of <django.core.servers.basehttp.WSGIServer object at 0x7f36ee0fcf10>>\n    \u2514 <Thread(Thread-1, started daemon 139873897260800)>\n  File \"/usr/lib/python3.8/socketserver.py\", line 650, in process_request_thread\n    self.finish_request(request, client_address)\n    \u2502    \u2502              \u2502        \u2514 ('127.0.0.1', 49952)\n    \u2502    \u2502              \u2514 <socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <function BaseServer.finish_request at 0x7f36f2b43e50>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f36ee0fcf10>\n  File \"/usr/lib/python3.8/socketserver.py\", line 360, in finish_request\n    self.RequestHandlerClass(request, client_address, self)\n    \u2502    \u2502                   \u2502        \u2502               \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f36ee0fcf10>\n    \u2502    \u2502                   \u2502        \u2514 ('127.0.0.1', 49952)\n    \u2502    \u2502                   \u2514 <socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <class 'django.core.servers.basehttp.WSGIRequestHandler'>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f36ee0fcf10>\n  File \"/usr/lib/python3.8/socketserver.py\", line 720, in __init__\n    self.handle()\n    \u2502    \u2514 <function WSGIRequestHandler.handle at 0x7f36efdbde50>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f36edf9da30>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 174, in handle\n    self.handle_one_request()\n    \u2502    \u2514 <function WSGIRequestHandler.handle_one_request at 0x7f36efdbdee0>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f36edf9da30>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 197, in handle_one_request\n    handler.run(self.server.get_app())\n    \u2502       \u2502   \u2502    \u2502      \u2514 <function WSGIServer.get_app at 0x7f36efdbd430>\n    \u2502       \u2502   \u2502    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f36ee0fcf10>\n    \u2502       \u2502   \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f36edf9da30>\n    \u2502       \u2514 <function BaseHandler.run at 0x7f36efe370d0>\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f36ee07c2b0>\n  File \"/usr/lib/python3.8/wsgiref/handlers.py\", line 137, in run\n    self.result = application(self.environ, self.start_response)\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2502    \u2514 <function BaseHandler.start_response at 0x7f36efe37430>\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f36ee07c2b0>\n    \u2502    \u2502        \u2502           \u2502    \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n    \u2502    \u2502        \u2502           \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f36ee07c2b0>\n    \u2502    \u2502        \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f36ee12dee0>\n    \u2502    \u2514 None\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f36ee07c2b0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/contrib/staticfiles/handlers.py\", line 76, in __call__\n    return self.application(environ, start_response)\n           \u2502    \u2502           \u2502        \u2514 <bound method BaseHandler.start_response of <django.core.servers.basehttp.ServerHandler object at 0x7f36ee07c2b0>>\n           \u2502    \u2502           \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n           \u2502    \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f36f352bbe0>\n           \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f36ee12dee0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/wsgi.py\", line 133, in __call__\n    response = self.get_response(request)\n               \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function BaseHandler.get_response at 0x7f36efe61f70>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f36f352bbe0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 130, in get_response\n    response = self._middleware_chain(request)\n               \u2502    \u2502                 \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f36f29859d0>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f36f352bbe0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f36f351dd90>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f36f2985a60>\n               \u2502           \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f36f351dd90>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f36f2988070>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f36f2985af0>\n               \u2502           \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f36f2988070>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.common.CommonMiddleware object at 0x7f36f29880d0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f36f2985b80>\n               \u2502           \u2514 <django.middleware.common.CommonMiddleware object at 0x7f36f29880d0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f36f2988100>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f36f2985c10>\n               \u2502           \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f36f2988100>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f36f2988310>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f36f2985dc0>\n               \u2502           \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f36f2988310>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f36f29882e0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f36f2985e50>\n               \u2502           \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f36f29882e0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f36f29883a0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function BaseHandler._get_response at 0x7f36f2985550>\n               \u2502           \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f36f29883a0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <bound method BaseHandler._get_response of <django.core.handlers.wsgi.WSGIHandler object at 0x7f36f352bbe0>>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 181, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n               \u2502                \u2502         \u2502                \u2514 {}\n               \u2502                \u2502         \u2514 ()\n               \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <function ClientRecordView at 0x7f36ef52f310>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 70, in view\n    return self.dispatch(request, *args, **kwargs)\n           \u2502    \u2502        \u2502         \u2502       \u2514 {}\n           \u2502    \u2502        \u2502         \u2514 ()\n           \u2502    \u2502        \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2502    \u2514 <function View.dispatch at 0x7f36f2996a60>\n           \u2514 <email_send.views.ClientRecordView object at 0x7f36ee0ce520>\n> File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 98, in dispatch\n    return handler(request, *args, **kwargs)\n           \u2502       \u2502         \u2502       \u2514 {}\n           \u2502       \u2502         \u2514 ()\n           \u2502       \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2514 <bound method ClientRecordView.post of <email_send.views.ClientRecordView object at 0x7f36ee0ce520>>\n  File \"/usr/lib/python3.8/contextlib.py\", line 75, in inner\n    return func(*args, **kwds)\n           \u2502     \u2502       \u2514 {}\n           \u2502     \u2514 (<email_send.views.ClientRecordView object at 0x7f36ee0ce520>, <WSGIRequest: POST '/contact/customer-record/'>)\n           \u2514 <function ClientRecordView.post at 0x7f36ee224dc0>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py\", line 56, in post\n    return render(request, 'error/add_record_clients.html')\n           \u2502      \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2514 <function render at 0x7f36ef57f670>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/shortcuts.py\", line 19, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n              \u2502      \u2502                \u2502              \u2502        \u2502              \u2514 None\n              \u2502      \u2502                \u2502              \u2502        \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n              \u2502      \u2502                \u2502              \u2514 None\n              \u2502      \u2502                \u2514 'error/add_record_clients.html'\n              \u2502      \u2514 <function render_to_string at 0x7f36f2d48310>\n              \u2514 <module 'django.template.loader' from '/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/templa...\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/template/loader.py\", line 61, in render_to_string\n    template = get_template(template_name, using=using)\n               \u2502            \u2502                    \u2514 None\n               \u2502            \u2514 'error/add_record_clients.html'\n               \u2514 <function get_template at 0x7f36f2d481f0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/template/loader.py\", line 19, in get_template\n    raise TemplateDoesNotExist(template_name, chain=chain)\n          \u2502                    \u2502                    \u2514 [TemplateDoesNotExist('error/add_record_clients.html')]\n          \u2502                    \u2514 'error/add_record_clients.html'\n          \u2514 <class 'django.template.exceptions.TemplateDoesNotExist'>\n\ndjango.template.exceptions.TemplateDoesNotExist: error/add_record_clients.html\n", "record": {"elapsed": {"repr": "0:00:27.349673", "seconds": 27.349673}, "exception": {"type": "TemplateDoesNotExist", "value": "error/add_record_clients.html", "traceback": true}, "extra": {}, "file": {"name": "base.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py"}, "function": "dispatch", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 98, "message": "An error has been caught in function 'dispatch', process 'MainProcess' (299448), thread 'Thread-1' (139873897260800):", "module": "base", "name": "django.views.generic.base", "process": {"id": 299448, "name": "MainProcess"}, "thread": {"id": 139873897260800, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:36:57.033549+00:00", "timestamp": 1620491817.033549}}}
{"text": "2021-05-08T16:38:39.898295+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:07.271701", "seconds": 7.271701}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 55, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 300446, "name": "MainProcess"}, "thread": {"id": 140125804930816, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:38:39.898295+00:00", "timestamp": 1620491919.898295}}}
{"text": "2021-05-08T16:39:09.264088+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:02.704271", "seconds": 2.704271}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 55, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 300735, "name": "MainProcess"}, "thread": {"id": 140397122262784, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:39:09.264088+00:00", "timestamp": 1620491949.264088}}}
{"text": "2021-05-08T16:40:09.233982+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:03.425539", "seconds": 3.425539}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 55, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 301153, "name": "MainProcess"}, "thread": {"id": 140640135968512, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:40:09.233982+00:00", "timestamp": 1620492009.233982}}}
{"text": "2021-05-08T16:40:12.606997+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:06.798554", "seconds": 6.798554}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 55, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 301153, "name": "MainProcess"}, "thread": {"id": 140640135968512, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:40:12.606997+00:00", "timestamp": 1620492012.606997}}}
{"text": "2021-05-08T16:40:26.223833+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:02.643813", "seconds": 2.643813}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 55, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 301276, "name": "MainProcess"}, "thread": {"id": 140157074204416, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:40:26.223833+00:00", "timestamp": 1620492026.223833}}}
{"text": "2021-05-08T16:40:29.110857+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:05.530837", "seconds": 5.530837}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 55, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 301276, "name": "MainProcess"}, "thread": {"id": 140157074204416, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:40:29.110857+00:00", "timestamp": 1620492029.110857}}}
{"text": "2021-05-08T16:40:51.669752+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:02.211302", "seconds": 2.211302}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 55, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 301440, "name": "MainProcess"}, "thread": {"id": 140480010344192, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:40:51.669752+00:00", "timestamp": 1620492051.669752}}}
{"text": "2021-05-08T16:41:11.952936+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:02.205539", "seconds": 2.205539}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 55, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 301689, "name": "MainProcess"}, "thread": {"id": 139634621282048, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:41:11.952936+00:00", "timestamp": 1620492071.952936}}}
{"text": "2021-05-08T16:41:33.663801+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:12.183050", "seconds": 12.18305}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 55, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 301806, "name": "MainProcess"}, "thread": {"id": 140441445304064, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:41:33.663801+00:00", "timestamp": 1620492093.663801}}}
{"text": "2021-05-08T16:45:56.752210+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:38.010987", "seconds": 38.010987}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 53, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 303608, "name": "MainProcess"}, "thread": {"id": 140000821090048, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:45:56.752210+00:00", "timestamp": 1620492356.75221}}}
{"text": "2021-05-08T16:46:37.758022+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:09.506372", "seconds": 9.506372}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 53, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 304086, "name": "MainProcess"}, "thread": {"id": 139925220370176, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:46:37.758022+00:00", "timestamp": 1620492397.758022}}}
{"text": "2021-05-08T16:47:32.916821+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:03.818538", "seconds": 3.818538}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 53, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 304560, "name": "MainProcess"}, "thread": {"id": 140169527797504, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:47:32.916821+00:00", "timestamp": 1620492452.916821}}}
{"text": "2021-05-08T16:47:32.917666+0000 ERROR An error has been caught in function 'dispatch', process 'MainProcess' (304560), thread 'Thread-1' (140169527797504):\nTraceback (most recent call last):\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py\", line 39, in post\n    client = Client.objects.get(user=request.user)\n             \u2502      \u2502                \u2502       \u2514 <SimpleLazyObject: <User: admin>>\n             \u2502      \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n             \u2502      \u2514 <django.db.models.manager.ManagerDescriptor object at 0x7f7bc8e74bb0>\n             \u2514 <class 'email_send.models.Client'>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/manager.py\", line 85, in manager_method\n    return getattr(self.get_queryset(), name)(*args, **kwargs)\n                   \u2502    \u2502               \u2502      \u2502       \u2514 {'user': <SimpleLazyObject: <User: admin>>}\n                   \u2502    \u2502               \u2502      \u2514 ()\n                   \u2502    \u2502               \u2514 'get'\n                   \u2502    \u2514 <function BaseManager.get_queryset at 0x7f7bcbafc550>\n                   \u2514 <django.db.models.manager.Manager object at 0x7f7bc8119370>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/db/models/query.py\", line 429, in get\n    raise self.model.DoesNotExist(\n          \u2502    \u2502     \u2514 <class 'email_send.models.Client.DoesNotExist'>\n          \u2502    \u2514 <class 'email_send.models.Client'>\n          \u2514 <QuerySet [<Client: \u041a\u043b\u0438\u0435\u043d\u0442 danil danil>]>\n\nemail_send.models.Client.DoesNotExist: Client matching query does not exist.\n\n\nDuring handling of the above exception, another exception occurred:\n\n\nTraceback (most recent call last):\n\n  File \"/usr/lib/python3.8/threading.py\", line 890, in _bootstrap\n    self._bootstrap_inner()\n    \u2502    \u2514 <function Thread._bootstrap_inner at 0x7f7bccd594c0>\n    \u2514 <Thread(Thread-1, started daemon 140169527797504)>\n  File \"/usr/lib/python3.8/threading.py\", line 932, in _bootstrap_inner\n    self.run()\n    \u2502    \u2514 <function Thread.run at 0x7f7bccd591f0>\n    \u2514 <Thread(Thread-1, started daemon 140169527797504)>\n  File \"/usr/lib/python3.8/threading.py\", line 870, in run\n    self._target(*self._args, **self._kwargs)\n    \u2502    \u2502        \u2502    \u2502        \u2502    \u2514 {}\n    \u2502    \u2502        \u2502    \u2502        \u2514 <Thread(Thread-1, started daemon 140169527797504)>\n    \u2502    \u2502        \u2502    \u2514 (<socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('...\n    \u2502    \u2502        \u2514 <Thread(Thread-1, started daemon 140169527797504)>\n    \u2502    \u2514 <bound method ThreadingMixIn.process_request_thread of <django.core.servers.basehttp.WSGIServer object at 0x7f7bc2ffccd0>>\n    \u2514 <Thread(Thread-1, started daemon 140169527797504)>\n  File \"/usr/lib/python3.8/socketserver.py\", line 650, in process_request_thread\n    self.finish_request(request, client_address)\n    \u2502    \u2502              \u2502        \u2514 ('127.0.0.1', 50190)\n    \u2502    \u2502              \u2514 <socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <function BaseServer.finish_request at 0x7f7bcba91e50>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f7bc2ffccd0>\n  File \"/usr/lib/python3.8/socketserver.py\", line 360, in finish_request\n    self.RequestHandlerClass(request, client_address, self)\n    \u2502    \u2502                   \u2502        \u2502               \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f7bc2ffccd0>\n    \u2502    \u2502                   \u2502        \u2514 ('127.0.0.1', 50190)\n    \u2502    \u2502                   \u2514 <socket.socket fd=6, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=('1...\n    \u2502    \u2514 <class 'django.core.servers.basehttp.WSGIRequestHandler'>\n    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f7bc2ffccd0>\n  File \"/usr/lib/python3.8/socketserver.py\", line 720, in __init__\n    self.handle()\n    \u2502    \u2514 <function WSGIRequestHandler.handle at 0x7f7bc8d0be50>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f7bc2f4df40>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 174, in handle\n    self.handle_one_request()\n    \u2502    \u2514 <function WSGIRequestHandler.handle_one_request at 0x7f7bc8d0bee0>\n    \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f7bc2f4df40>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/servers/basehttp.py\", line 197, in handle_one_request\n    handler.run(self.server.get_app())\n    \u2502       \u2502   \u2502    \u2502      \u2514 <function WSGIServer.get_app at 0x7f7bc8d0b430>\n    \u2502       \u2502   \u2502    \u2514 <django.core.servers.basehttp.WSGIServer object at 0x7f7bc2ffccd0>\n    \u2502       \u2502   \u2514 <django.core.servers.basehttp.WSGIRequestHandler object at 0x7f7bc2f4df40>\n    \u2502       \u2514 <function BaseHandler.run at 0x7f7bc8d850d0>\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f7bc805b100>\n  File \"/usr/lib/python3.8/wsgiref/handlers.py\", line 137, in run\n    self.result = application(self.environ, self.start_response)\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2502    \u2514 <function BaseHandler.start_response at 0x7f7bc8d85430>\n    \u2502    \u2502        \u2502           \u2502    \u2502        \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f7bc805b100>\n    \u2502    \u2502        \u2502           \u2502    \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n    \u2502    \u2502        \u2502           \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f7bc805b100>\n    \u2502    \u2502        \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f7bc85003d0>\n    \u2502    \u2514 None\n    \u2514 <django.core.servers.basehttp.ServerHandler object at 0x7f7bc805b100>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/contrib/staticfiles/handlers.py\", line 76, in __call__\n    return self.application(environ, start_response)\n           \u2502    \u2502           \u2502        \u2514 <bound method BaseHandler.start_response of <django.core.servers.basehttp.ServerHandler object at 0x7f7bc805b100>>\n           \u2502    \u2502           \u2514 {'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/proger:@/tmp/.ICE-unix/2064,unix/proger:/tmp/.ICE-unix/2064', 'QT_ACCESSIBIL...\n           \u2502    \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f7bcc47abe0>\n           \u2514 <django.contrib.staticfiles.handlers.StaticFilesHandler object at 0x7f7bc85003d0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/wsgi.py\", line 133, in __call__\n    response = self.get_response(request)\n               \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function BaseHandler.get_response at 0x7f7bc8daff70>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f7bcc47abe0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 130, in get_response\n    response = self._middleware_chain(request)\n               \u2502    \u2502                 \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f7bcb8d39d0>\n               \u2514 <django.core.handlers.wsgi.WSGIHandler object at 0x7f7bcc47abe0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f7bcc46bd90>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f7bcb8d3a60>\n               \u2502           \u2514 <django.middleware.security.SecurityMiddleware object at 0x7f7bcc46bd90>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f7bcb8d6070>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f7bcb8d3af0>\n               \u2502           \u2514 <django.contrib.sessions.middleware.SessionMiddleware object at 0x7f7bcb8d6070>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.common.CommonMiddleware object at 0x7f7bcb8d60d0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f7bcb8d3b80>\n               \u2502           \u2514 <django.middleware.common.CommonMiddleware object at 0x7f7bcb8d60d0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f7bcb8d6100>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f7bcb8d3c10>\n               \u2502           \u2514 <django.middleware.csrf.CsrfViewMiddleware object at 0x7f7bcb8d6100>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f7bcb8d6220>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f7bcb8d3dc0>\n               \u2502           \u2514 <django.contrib.auth.middleware.AuthenticationMiddleware object at 0x7f7bcb8d6220>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f7bcb8d62e0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function convert_exception_to_response.<locals>.inner at 0x7f7bcb8d3e50>\n               \u2502           \u2514 <django.contrib.messages.middleware.MessageMiddleware object at 0x7f7bcb8d62e0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f7bcb8d63a0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/utils/deprecation.py\", line 114, in __call__\n    response = response or self.get_response(request)\n               \u2502           \u2502    \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2502           \u2502    \u2514 <function BaseHandler._get_response at 0x7f7bcb8d3550>\n               \u2502           \u2514 <django.middleware.clickjacking.XFrameOptionsMiddleware object at 0x7f7bcb8d63a0>\n               \u2514 None\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n               \u2502            \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <bound method BaseHandler._get_response of <django.core.handlers.wsgi.WSGIHandler object at 0x7f7bcc47abe0>>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 181, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n               \u2502                \u2502         \u2502                \u2514 {}\n               \u2502                \u2502         \u2514 ()\n               \u2502                \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n               \u2514 <function ClientRecordView at 0x7f7bc847d310>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 70, in view\n    return self.dispatch(request, *args, **kwargs)\n           \u2502    \u2502        \u2502         \u2502       \u2514 {}\n           \u2502    \u2502        \u2502         \u2514 ()\n           \u2502    \u2502        \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2502    \u2514 <function View.dispatch at 0x7f7bcb8e3a60>\n           \u2514 <email_send.views.ClientRecordView object at 0x7f7bc3008c10>\n> File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py\", line 98, in dispatch\n    return handler(request, *args, **kwargs)\n           \u2502       \u2502         \u2502       \u2514 {}\n           \u2502       \u2502         \u2514 ()\n           \u2502       \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2514 <bound method ClientRecordView.post of <email_send.views.ClientRecordView object at 0x7f7bc3008c10>>\n  File \"/usr/lib/python3.8/contextlib.py\", line 75, in inner\n    return func(*args, **kwds)\n           \u2502     \u2502       \u2514 {}\n           \u2502     \u2514 (<email_send.views.ClientRecordView object at 0x7f7bc3008c10>, <WSGIRequest: POST '/contact/customer-record/'>)\n           \u2514 <function ClientRecordView.post at 0x7f7bc8155dc0>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py\", line 54, in post\n    return render(request, '<h1>\u041e\u0448\u0438\u0431\u043a\u0430</h1>')\n           \u2502      \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n           \u2514 <function render at 0x7f7bc84cc670>\n\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/shortcuts.py\", line 19, in render\n    content = loader.render_to_string(template_name, context, request, using=using)\n              \u2502      \u2502                \u2502              \u2502        \u2502              \u2514 None\n              \u2502      \u2502                \u2502              \u2502        \u2514 <WSGIRequest: POST '/contact/customer-record/'>\n              \u2502      \u2502                \u2502              \u2514 None\n              \u2502      \u2502                \u2514 '<h1>\u041e\u0448\u0438\u0431\u043a\u0430</h1>'\n              \u2502      \u2514 <function render_to_string at 0x7f7bcbc96310>\n              \u2514 <module 'django.template.loader' from '/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/templa...\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/template/loader.py\", line 61, in render_to_string\n    template = get_template(template_name, using=using)\n               \u2502            \u2502                    \u2514 None\n               \u2502            \u2514 '<h1>\u041e\u0448\u0438\u0431\u043a\u0430</h1>'\n               \u2514 <function get_template at 0x7f7bcbc961f0>\n  File \"/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/template/loader.py\", line 19, in get_template\n    raise TemplateDoesNotExist(template_name, chain=chain)\n          \u2502                    \u2502                    \u2514 [TemplateDoesNotExist('<h1>\u041e\u0448\u0438\u0431\u043a\u0430</h1>')]\n          \u2502                    \u2514 '<h1>\u041e\u0448\u0438\u0431\u043a\u0430</h1>'\n          \u2514 <class 'django.template.exceptions.TemplateDoesNotExist'>\n\ndjango.template.exceptions.TemplateDoesNotExist: <h1>\u041e\u0448\u0438\u0431\u043a\u0430</h1>\n", "record": {"elapsed": {"repr": "0:00:03.819383", "seconds": 3.819383}, "exception": {"type": "TemplateDoesNotExist", "value": "<h1>\u041e\u0448\u0438\u0431\u043a\u0430</h1>", "traceback": true}, "extra": {}, "file": {"name": "base.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/venv/lib/python3.8/site-packages/django/views/generic/base.py"}, "function": "dispatch", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 98, "message": "An error has been caught in function 'dispatch', process 'MainProcess' (304560), thread 'Thread-1' (140169527797504):", "module": "base", "name": "django.views.generic.base", "process": {"id": 304560, "name": "MainProcess"}, "thread": {"id": 140169527797504, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:47:32.917666+00:00", "timestamp": 1620492452.917666}}}
{"text": "2021-05-08T16:47:46.183740+0000 ERROR Error! not client Client matching query does not exist.\n", "record": {"elapsed": {"repr": "0:00:01.308412", "seconds": 1.308412}, "exception": null, "extra": {}, "file": {"name": "views.py", "path": "/home/proger/\u0414\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u044b/tattoo_studio_dj/tattoo/email_send/views.py"}, "function": "post", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 53, "message": "Error! not client Client matching query does not exist.", "module": "views", "name": "email_send.views", "process": {"id": 304762, "name": "MainProcess"}, "thread": {"id": 139956620642048, "name": "Thread-1"}, "time": {"repr": "2021-05-08 16:47:46.183740+00:00", "timestamp": 1620492466.18374}}}
